lab8.md

```diff
@@ -0,0 +1,64 @@
+# Lab 8 - Auto-scaling in action
+
+<img src="https://github.com/openfaas/media/raw/master/OpenFaaS_Magnet_3_1_png.png" width="500px"></img>
+
+As described [here](http://docs.openfaas.com/architecture/autoscaling/) in the documentation Open FaaS ships with auto-scaling in this lab we will have a look how the auto-scaling works in action.
+
+## Pre-requisites:
+
+* At this point, it is expected you already have set up Open FaaS for Docker Swarm as described in [Lab 1](https://github.com/openfaas/workshop/blob/master/lab1.md).
+
+* As we need a way to invoke our lambda to demonstrate auto-scaling it is expected that you have access to [Curl](http://macappstore.org/curl/) or another to which allows you to do an HTTP-Request.
+
+
+## Exploring Auto-Scaling:
+
+Before dive into a hands-on session, we will have a brief look how we can configure the auto-scaling out of the box. 
+
+Out of the box Open FaaS is set up in a way that it will auto-scale based on the received `request per seconds`. If the defined threshold for the `request per seconds` is exceeded the AlertManager will fire. 
+
+> If you want the scaling happening based on CPU and Memory usage please have a brief look at [this blog](https://stefanprodan.com/2018/kubernetes-scaleway-baremetal-arm-terraform-installer/#horizontal-pod-autoscaling)
+
+For each fired alarm the auto-scaler will add 5 instances to the affected lambda. Open FaaS has two configuration options that allow to specify the starting/minimum amount of instances and also allows to ceil the maximum amount of instances:
+
+You can control the minimum amount of instances for lambda by setting `com.openfaas.scale.min`, the default value is currently `1`. 
+
+You can control the maximum amount of instances that can spawn for a lambda by setting `com.openfaas.scale.max`, the default value is currently `20`. 
+
+> Note: If you set `com.openfaas.scale.min` and `com.openfaas.scale.max` to the same value you are disabling the auto-scaling feature. So pay attention to not do this accidentally. 
+
+## Hands-on:
+
+### Preparing Prometheus:
+
```

Comment:
Please assume the port is 9090 and remove this section up until line 40 where you ask them to open a browser.